setup.py

```diff
@@ -19,8 +19,20 @@
     include_package_data=True,
     zip_safe=False,
     classifiers=[
-        'Programming Language :: Python :: 3',
+        'Development Status :: 4 - Beta',
+        'Framework :: Sphinx',
+        'Framework :: Sphinx :: Extension',
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2.7',
```

Comment:
I don't think we should support python 2 for new extensions that are optional for building on rtd

Reply:
Yeah. It's currently supported, though.

We can remove it once we remove it from `tox` on Jan 1st :)